### DISCUSSION TITLE
No warning about removing my own access rights

### DISCUSSION THREAD

**Discussion by robpridham** [community/community#37195] [Repositories]
This is a feature request / request for improvement. If this is better raised somewhere else, please direct me accordingly.

Yesterday I ran into a problem. I wanted to reduce the breadth of admin permissions for a team, which I am a member of. My intention was to reduce the team's role to from 'Admin' to 'Maintainer' and then add myself as an individual admin.

Of course, not thinking about this, as soon as I set the team's dropdown to 'Maintainer', it applied immediately and removed my own access to make further changes.

A confirmation prompt warning that I was removing my own access would have been helpful.

**Comment by ossanna16** [comment]
Hi there @robpridham-bbc and welcome to our community! Thank you for your feature request 🙂
